src/server/services/wfs/qgswfsgetfeature.cpp

```diff
@@ -1528,22 +1521,38 @@ namespace QgsWfs
           continue;
         }
 
-        const QgsField field = fields.at( idx );
-        const QgsEditorWidgetSetup setup = field.editorWidgetSetup();
+        QDomElement fieldElem = createFieldElement( fields.at( idx ), idx, featureAttributes, doc );
+        typeNameElement.appendChild( fieldElem );
+      }
+
+      return featureElement;
+    }
 
-        QString attributeName = field.name();
+    QDomElement createFieldElement( const QgsField &field, const int idx, const QgsAttributes &featureAttributes, QDomDocument &doc )
+    {
+      const QgsEditorWidgetSetup setup = field.editorWidgetSetup();
+      QString attributeName = field.name();
```

Comment:
const

Reply:
There are 2 `replace` the line just after